packages/celotool/src/lib/migration-utils.ts

```diff
@@ -0,0 +1,51 @@
+import { envVar, fetchEnv } from './env-utils'
+import {
+  AccountType,
+  generatePrivateKey,
+  getAddressesFor,
+  getPrivateKeysFor,
+  privateKeyToAddress,
+} from './generate_utils'
+import { OG_ACCOUNTS } from './genesis_constants'
+import { ensure0x } from './utils'
+
+export function minerForEnv() {
+  if (fetchEnv(envVar.VALIDATORS) === 'og') {
```

Comment:
We don't use the 'og' setup anymore so this might be a good opportunity to clean up some tech debt and remove this

Reply:
I'm going to leave it in simply because I don't know what to do about https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/helm_deploy.ts#L517-L521